testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/group/GroupTest.java

```diff
@@ -160,27 +160,6 @@ public void testClientRemoveWithClientRoleGroupMapping() throws Exception {
         assertAdminEvents.assertEvent(testRealmId, OperationType.DELETE, AdminEventPaths.clientResourcePath(clientUuid), ResourceType.CLIENT);
     }
 
-    private GroupRepresentation createGroup(RealmResource realm, GroupRepresentation group) {
```

Comment:
Why is this removed? Is this somehow relevant to this PR?

Reply:
I took the liberty of moving this to AbstractGroupTest to prevent code duplication

Reply:
Ah, I missed that. That makes sense